### ISSUE TITLE
Request for Fixing Motion Issues and Mod Potion Effects in the Next Update

### ISSUE THREAD

**Issue by INV-PLA** [TheEpicBlock/PolyMc#226]
Can the next update fix the issue of motion not actually changing between the server and client, as well as the problem where mod potion effects on the server may be considered as prohibited by the mini-map functionality?

**Comment by TheEpicBlock** [comment]
> the problem where mod potion effects on the server may be considered as prohibited by the mini-map functionality?

What does this mean exactly? I know certain minimap mods use potion effect to signal functionality should be blocked, but surely that's not what's happening here?

**Comment by fennecr** [comment]
I've noticed this bug as well, if a player has a minimap installed but no other mods installed on their client and they receive custom potion effects, they get their minimap or other functionality disabled. Not really a polymc issue though as it seems like the minimap mod is just interpreting it wrong. 
My fix is just to install the mods clientside along with the minimap so the potion effects are correctly identified.

**Comment by INV-PLA** [comment]
There is another problem, some modules on the server will not take effect on the client if you modify motion. Can you fix it?

**Comment by INV-PLA** [comment]
> Maybe that would work, but I mean installing the mods that add potion effects onto the client if you're using a mini-map. I figure anyone who is already using mods shouldn't have too hard of a time adding a few more.

The result will cause others to not go to the server test if the server installed map

**Comment by TheEpicBlock** [comment]
Which minimap mod is being used exactly? I know Xaero's uses potions but I don't think think it's intended to disable everything for every unrecognized potion

**Comment by INV-PLA** [comment]
yes Xaero's The main thing is that he detects a digital ID, and if the ID matches, it will think that it is the potion that comes with the module

> Which minimap mod is being used exactly? I know Xaero's uses potions but I don't think think it's intended to disable everything for every unrecognized potion
